### PR TITLE
Add logo to imported events

### PR DESCRIPTION
**Pull request by tarnap**
By giving the logo as an optional argument, each batch can have its own
logo. This allow to add different logos for different calendars.

import with logo

See also issue 114 on veranstaltungen_winterthur

### CODE REVIEW

**Comment by tarnap** [comment]
@mbaechtold @maethu ping :)

**Comment by maethu** [comment]
pong

File: eventary/management/commands/importcalendar.py [hunk 1]
```diff
@@ -126,6 +137,10 @@ def handle_event_data(self, event_data):
             recurring='recurringEventId' in event_data,
         )
 
+        # set the image
+        with open(self.options.get('logo'), 'rb') as image:
```
Comment (maethu) [inline eventary/management/commands/importcalendar.py]: I would suggest only one small improvement. As I see the image is scratched from disk for every single imported event. 

This is not necessary and could be improved by only load the image once per command.

Comment (tarnap) [inline eventary/management/commands/importcalendar.py]: :+1:

Comment (tarnap) [inline eventary/management/commands/importcalendar.py]: @maethu The logo is now opened only once. Thanks for the improvement!

**Pull request merged**